src/scheme/bit2.rs

```diff
@@ -193,7 +194,9 @@ impl<R: Rng + SeedableRng> ORECipher for OreAes128<R> {
         let b_f = &b[num_blocks..];
 
         for n in 0..num_blocks {
-            if a[n] != b[n] || left_block(a_f, n) != left_block(b_f, n) {
+            if a[n] != b[n]
```

Comment:
Do we need to use `Choice` to wrap these as well? if `a[n]` is not equal to `b[n]` the whole thing will exit early, right?

Reply:
Yes we do - good catch